packages/uhk-common/src/util/constants.ts

```diff
@@ -2,5 +2,6 @@ export namespace Constants {
     export const AGENT_GITHUB_URL = 'https://github.com/UltimateHackingKeyboard/agent';
     export const AGENT_CONTRIBUTORS_GITHUB_PAGE_URL = 'https://github.com/UltimateHackingKeyboard/agent/graphs/contributors';
     export const AGENT_CONTRIBUTORS_GITHUB_API_URL = 'https://api.github.com/repos/UltimateHackingKeyboard/agent/contributors';
+    export const AGENT_HELP_ACCENT_CHARS_URL = 'https://ultimatehackingkeyboard.com/blog/2018/06/23/how-can-i-type-accented-characters-with-my-uhk'
```

Comment:
This constant is never used. Why did you add it?

Reply:
It is in the help page.  It was originally embedded directly in the HTML of the component.  I didn't change it in this original commit/PR, but it's should be used now.  It's discussed in comments of the original issue thread.